File: lab1/CourseResult.java
public class CourseResult {
    public String studentName;
    public String courseName;
    public String grade;

    public void display() {
        System.out.println("Student Name:" + studentName);
        System.out.println("Course Name:" + courseName);
        System.out.println("Grade:" + grade);
    }

    public static void main(String[] args) {
        CourseResult cr1 = new CourseResult();
        cr1.studentName = "Jawad Ahmad";
        cr1.courseName = "OOP";
        cr1.grade = "A+";
        cr1.display();
        CourseResult cr2 = new CourseResult();
        cr2.studentName = "Ahmad";
        cr2.courseName = "Java";
        cr2.grade = "A";
        cr2.display();
    }
}

File: lab1/CourseResult.cpp
#include <iostream>
#include <string>
using namespace std;
struct CourseResult {
    string studentName;   
    string courseName;
    string grade;
    void display() {
        cout << "Student Name: " << studentName << endl;
        cout << "Course Name: " << courseName << endl;
        cout << "Grade: " << grade << endl;
    }
};

int main() {
    CourseResult cr1; 
    cr1.studentName = "Jawad Ahmad";
    cr1.courseName = "OOP";
    cr1.grade = "A+";
    cr1.display();
    cr2.studentName = "Ahmad";
    cr2.courseName = "JAVA";
    cr2.grade = "A";
    cr2.display();
}